src/frontend/src/components/Availabilities.tsx

```diff
@@ -0,0 +1,102 @@
+import { Grid, Typography } from '@mui/material';
+import { Box } from '@mui/system';
+import { User } from 'shared';
+import { EnumToArray, HeatmapColors } from '../utils/design-review.utils';
+import { fullNamePipe } from '../utils/pipes';
+import NERFailButton from './NERFailButton';
+import NERSuccessButton from './NERSuccessButton';
+import WarningIcon from '@mui/icons-material/Warning';
+
+interface AvailabilitiesProps {
+  currentAvailableUsers: User[];
+  currentUnavailableUsers: User[];
+  usersToAvailabilities: Map<User, number[]>;
+}
+
+const Availabilities: React.FC<AvailabilitiesProps> = ({
+  currentAvailableUsers,
+  currentUnavailableUsers,
+  usersToAvailabilities
+}) => {
+  const totalUsers = usersToAvailabilities.size;
+  const fontSize = totalUsers > 10 ? '1em' : totalUsers > 15 ? '0.8em' : '1.2em';
+  const colors = EnumToArray(HeatmapColors);
+
+  return (
+    <Grid
+      style={{
+        display: 'flex',
+        flexDirection: 'column',
+        backgroundColor: '#2C2C2C',
+        padding: '20px',
+        borderRadius: '8px',
+        height: '100%',
+        overflow: 'auto'
+      }}
+    >
+      <Grid style={{ display: 'flex', flexDirection: 'column', flexGrow: 1, marginBottom: '10px' }}>
+        <Grid sx={{ display: 'flex', justifyContent: 'center' }}>
+          <Typography style={{ marginRight: '10px' }}>0/0</Typography>
+          {Array.from({ length: 6 }, (_, i) => (
```

Comment:
color legend isn't showing up for me:
<img width="212" alt="image" src="https://github.com/Northeastern-Electric-Racing/FinishLine/assets/29521172/c64ca453-bbf6-4090-86d5-a536671b254b">